tests/test_certification.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+# Packages
+from vcr_unittest import VCRTestCase
+
+# Local
+from webapp.app import app
+
+
+class TestCertification(VCRTestCase):
+    def _get_vcr_kwargs(self):
+        """
+        This removes the authorization header
+        from VCR so we don't record auth parameters
+        """
+        return {
+            "decode_compressed_response": True,
+            "filter_headers": ["Authorization", "Cookie"],
+        }
+
+    def setUp(self):
+        app.testing = True
+        self.client = app.test_client()
+        return super().setUp()
+
+    def test_certification_home(self):
+        response = self.client.get("/certification")
+        self.assertEqual(response.status_code, 200)
+
+    def test_home_api_endpoint(self):
```

Comment:
Is this test really needed? Once the you move the logic into the home view `test_certification_home` should fail if the response returned from the API is not enough to build the homepage. I also can't seem to spot where in `CertificationAPI` do we need that key for it to work properly , am I missing something?

Reply:
I think it's a good first example test for future calls to the API but I could do it in the next PR.
Yes that key is in line 75 of `api.py`

Reply:
On line `75` you are just passing `certifiedmodels` as a parameter to `_get` so it can build the URL. I don't think at any point the class requires `certifiedmodels` to be in the response payload to function.

Reply:
Removed.